python/py_helpers.test.py

```diff
@@ -254,6 +310,46 @@ def test_find_body_without_body(self):
 
         self.assertEqual(node.find_variable("x").find_body(), Node())
 
+    def test_inherits_from(self):
+        code_str = """
+class A:
+   pass
+   
+class B(A, C):
+   pass
+"""
+        node = Node(code_str)
+
+        self.assertFalse(node.find_class("A").inherits_from("B"))
+        self.assertTrue(node.find_class("B").inherits_from("C", "A"))
+
+    def test_find_method_args(self):
+        code_str = """
+class A:
+   def __init__(self, *, a, b=0):
+     pass
+"""
+        node = Node(code_str)
+        
+        self.assertTrue(node.find_class("A").find_function("__init__").has_args("self, *, a, b=0"))
+
+    def test_has_decorators(self):
+        code_str = """
+class A:
+  @property
+  @staticmethod
+  def foo():
+    pass
+    
+  def bar():
+    pass    
+"""
+        node = Node(code_str)
+
+        self.assertTrue(node.find_class("A").find_function("foo").has_decorators("property", "staticmethod"))
+        self.assertFalse(node.find_class("A").find_function("foo").has_decorators("property"))
```

Comment:
Is this the desired behaviour? It seems a little counter-intuitive to me, since A.foo does have `@property`

Reply:
You're right, it's a bit counter-intuitive. Would it be better to check if a specific decorator (or multiple decorators) is present and avoid to check that the number of arguments passed is the same of `len(self.tree.decorator_list)`?

Reply:
That makes sense to me. So, for this case I think the following should be the tests:

```py
has_decorators("property", "staticmethod") # True
has_decorators("property", "staticmethod", "anything") # False
has_decorators("property") # True
```